NugetLightRequest.cs

```diff
@@ -1909,5 +1929,134 @@ private IEnumerable<PackageItem> SearchForPackages(PackageSource source, string
                 return Enumerable.Empty<PackageItem>();
             }
         }
+
+        internal NetworkCredential GetCredsFromCredProvider(string query, NuGetRequest request, bool isRetry=false)
```

Comment:
>GetCredsFromCredProvider [](start = 35, length = 24)

Today both az cli SDK and Azure PowerShell cmdlets will display failure if a user has not logon to Azure. A user has to login to Azure first, then az and Azure cmdlets can work. Can we consider the same behavior?  Find-package will fail and ask for login to Azure. A user run the tool, and find-package again will pass.